src/main/java/com/cb/controllers/ShopController.java

```diff
@@ -0,0 +1,84 @@
+package com.cb.controllers;
+
+import com.cb.dto.DefaultDTO;
+import com.cb.services.dbService.dbServiceImpl.ShopDBServiceImpl;
+import com.cb.services.dbService.iDbService.FighterDBService;
+import com.cb.services.mapService.iMapService.ShopService;
+import com.cb.services.mapService.iMapService.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+
+@Controller
+public class ShopController {
+
+    @Autowired
+    ShopService shopService;
+
+    @Autowired
+    UserService userService;
+
+    @Autowired
+    FighterDBService fighterDBService;
+    @Autowired
+    ShopDBServiceImpl shopDBService;
+
+    @RequestMapping(method = RequestMethod.GET, value = "/shop")
+    public String getWeaponsTable(Model m, HttpServletRequest req) {
+
+        HttpSession userSession = req.getSession();
+        int userid = (int) userSession.getAttribute("id");
+        String username = (String) userSession.getAttribute("userName");
```

Comment:
Variable names should be written in convension:

If the name you choose consists of only one word, spell that word in all lowercase letters. If it consists of more than one word, capitalize the first letter of each subsequent word. The names gearRatio and currentGear are prime examples of this convention. If your variable stores a constant value, such as static final int NUM_GEARS = 6, the convention changes slightly, capitalizing every letter and separating subsequent words with the underscore character. By convention, the underscore character is never used elsewhere.

Reply:
ok changed it